python/pyspark/pandas/frame.py

```diff
@@ -5317,6 +5317,12 @@ def to_orc(
         ...     '%s/to_orc/foo.orc' % path,
         ...     mode = 'overwrite',
         ...     partition_cols=['date', 'country'])
+
+        Notes
+        -----
+        pandas-on-Spark writes ORC files into the directory, `path`, and writes
+        multiple `part-...` files in the directory unlike pandas.
+        pandas-on-Spark respects HDFS's property such as 'fs.default.name'.
```

Comment:
Do we have this note in other I/O API? it applies to all I/O API in pandas api on spark.

Reply:
Yes, just also applied to `to_parquet`.

Checked it's already noted in other I/O APIs!